sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
```

Comment:
I haven't the read the contract on the key. Don't know if it is supposed to stay static or not. Will let you guys decide.